cityofaustin/atd-moped
# Show a delete confirmation modal when deleting task orders

## Associated issues
Fixes cityofaustin/atd-data-tech/issues/11069

Shoutout to @tillyw for the wonderful code I borrowed :)

## Testing
**URL to test:** 

https://deploy-preview-923--atd-moped-main.netlify.app/moped/projects

**Steps to test:**
1. Go to the funding page of a project
2. Add task orders
3. Upon trying to delete a task order you should get a delete confirmation modal

---
#### Ship list
